src/Emogrifier.php

```diff
@@ -1404,18 +1420,13 @@ private function getCssSelectorPrecedence($selector)
         $selectorKey = md5($selector);
         if (!isset($this->caches[static::CACHE_KEY_SELECTOR][$selectorKey])) {
             $precedence = 0;
-            $value = 100;
-            // ids: worth 100, classes: worth 10, elements: worth 1
-            $search = ['\\#', '\\.', ''];
-
-            foreach ($search as $s) {
+            foreach ($this->selectorPrecedenceMatchers as $s => $value) {
```

Comment:
While we're touching the foreach anyway, please let's rename `$s` to something more expressive, e.g. `$matcher`.

Reply:
Done